novaagent/libs/centos.py

```diff
@@ -198,11 +207,11 @@ def is_network_manager(self):
             out, err = p.communicate()
             if p.returncode != 0:
                 return False
-            if 'enabled' in out:
+            if b'enabled' in out:
```

Comment:
I don't think this will work consistently between PY2/PY3.  A better approach would be:

```suggestion
            if 'enabled'.encode() in out:
```

That will use `'enabled'` on PY2 and `b'enabled'` on PY3.

Reply:
Updated.